notifications/rest/bindings/create-binding/create-binding.php

```diff
@@ -0,0 +1,17 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require './vendor/autoload.php';
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
+$token = "{{ auth_token }}"; 
+
+// Initialize the client
+$client = new Twilio\Rest\Client($sid, $token);
+
+$service = $client->notifications->v1->services("ISxxx");
+
+$binding = $service->bindings->create("xxx", "bob", "gcm", "xxx", array("tag" => "premium"));
+
+print print_r($binding, true);
```

Comment:
@jefflinwood Any reason to choose `print print_r($binding, true);` over `print_r($binding);`?